tests/integration/prometheus.py

```diff
@@ -81,4 +88,4 @@ async def check_metrics(self, query: str):
         assert response.code == 200, f"Failed to query '{query}': {data}"
         result = json.loads(data)
         assert result.get("status") == "success", f"Query failed: {result}"
-        assert result.get("data", {}).get("result"), "Data not yet available"
+        return result.get("data", {}).get("result", [])
```

Comment:
Let's do the checking outside of this method.  An invalid response is okay to check -- but really i wanna know if I got empty data